spec/requests/api/authentications_spec.rb

```diff
@@ -71,6 +71,67 @@
       run_post(authentications_url, :action => 'delete', :resources => [{ 'id' => auth.id }])
       expect(response).to have_http_status(:forbidden)
     end
+
+    it 'requires a type to create an authentication' do
+      api_basic_authorize collection_action_identifier(:authentications, :create, :post)
+
+      run_post(authentications_url, :action => 'create', :name => 'foo')
+
+      expected = {
+        'error' => a_hash_including(
+          'message' => 'Cannot create Authentication - must supply type'
+        )
+      }
+      expect(response).to have_http_status(:bad_request)
+      expect(response.parsed_body).to include(expected)
+    end
+
+    it 'requires a manager when creating an authentication' do
+      api_basic_authorize collection_action_identifier(:authentications, :create, :post)
+
+      run_post(authentications_url, :action => 'create', :type => 'Authentication')
+
+      expected = {
+        'error' => a_hash_including(
+          'message' => 'Cannot create Authentication - must supply a manager'
+        )
+      }
+      expect(response).to have_http_status(:bad_request)
+      expect(response.parsed_body).to include(expected)
+    end
+
+    it 'requires that the type support create_in_provider_queue' do
+      api_basic_authorize collection_action_identifier(:authentications, :create, :post)
+
+      run_post(authentications_url, :action => 'create', :type => 'Authentication', :manager => 'blah')
```

Comment:
I see, type is 'Authentication',   should we default to this type if not specified ? (it is /api/authentications after all)